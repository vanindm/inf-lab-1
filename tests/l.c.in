4 2
0.05719,0.00263 0.0687,0.04085 
0.05177,0.04213 0.06328,0.081 
0.08838,0.00137 0.0077,0.07192 
0.07302,0.0042 0.0008,0.04484 
2
1,0 0,0 0.5,0.5
