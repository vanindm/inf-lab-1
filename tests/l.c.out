0.057190,0.002630 0.068700,0.040850 
0.051770,0.042130 0.063280,0.081000 
0.179980,0.042610 0.054380,0.135590 
0.073020,0.004200 0.000800,0.044840 
